internal/terminal/flags.go

```diff
@@ -50,3 +51,27 @@ func isValidOutputFormat(outputFormat OutputFormat) bool {
 	}
 	return false
 }
+
+// OutputTarget is the terminal output target
+type OutputTarget string
+
+// Path returns the output target filepath
+func (ot OutputTarget) Path() string { return string(ot) }
+
+// String returns the output target display
+func (ot OutputTarget) String() string {
+	val := ot.Path()
+	if val == "" {
+		return "<none>"
```

Comment:
should "<none>" be a const?

Reply:
you raise a good point, in that this type isn't needed anymore.  going to remove it in favor of just a string flag